app/src/ereferrals/java/org/eyeseetea/malariacare/layout/adapters/dashboard/strategies/DashboardAdapterStrategy.java

```diff
@@ -95,5 +101,17 @@ public void renderSurveySummary(View rowView, SurveyDB survey) {
     public boolean hasAllComplementarySurveys(SurveyDB malariaSurvey) {
         return true;
     }
+
+    private boolean hasPhone(SurveyDB survey) {
```

Comment:
@manuelplazaspalacio maybe we could improve the performance of this by using a join? Is there any reason for not doing it?